common/web/gesture-recognizer/src/engine/headless/gestures/specs/contactModel.ts

```diff
@@ -38,5 +30,18 @@ export interface ContactModel {
   // Allows specifying use of one of the configured 'gestureItemIdentifiers'.
   recognizerId?: string;
 
-  readonly onItemChange?: 'reject' | 'resolve'; // may be undefined for 'continue'
+  /**
+   * Used to either instantly resolve or reject this component of a gesture based on
+   * a change in 'currently hovered item' for the path.
+   *
+   * For example, a longpress component should likely reject on change of base item,
+   * allowing a new longpress (with a fresh timer) to start with the new base item.
+   */
+  readonly itemChangeAction?: 'reject' | 'resolve'; // may be undefined for 'continue'
+
+  // Is needed to define whether or not the contact-point should be ignored by this gesture type.
+  // If undefined, defaults to () => true.
+  // Param 2 only matters for slot 2+ of a multitouch gesture spec; it will receive the latest sample
+  // from already-linked.
```

Comment:
Should this be a javadoc comment?

```suggestion
  /**
   * Is needed to define whether or not the contact-point should be ignored by this gesture type.
   * If undefined, defaults to () => true.
   * Param 2 only matters for slot 2+ of a multitouch gesture spec; it will receive the latest sample
   * from already-linked.
   */
```